subworkflows/local/fastqc_trimgalore.nf

```diff
@@ -0,0 +1,52 @@
+//
+// Read QC, UMI extraction and trimming
+//
+
+params.fastqc_options     = [:]
+params.trimgalore_options = [:]
+
+include { FASTQC           } from '../../modules/nf-core/fastqc/main'           addParams( options: params.fastqc_options     )
+include { TRIMGALORE       } from '../../modules/nf-core/trimgalore/main'       addParams( options: params.trimgalore_options )
+
+workflow FASTQC_TRIMGALORE {
+    take:
+    reads         // channel: [ val(meta), [ reads ] ]
+    skip_fastqc   // boolean: true/false
+    skip_trimming // boolean: true/false
+
+    main:
+    ch_versions    = Channel.empty()
+    fastqc_html    = Channel.empty()
+    fastqc_zip     = Channel.empty()
```

Comment:
Here the indentation is intuitive to me, so indeed, seems good to me to make indentation uniform.